api/src/main/java/io/opentelemetry/trace/DefaultTracer.java

```diff
@@ -172,7 +172,7 @@ private NoopSpanBuilder(Tracer tracer, String name) {
     @Override
     public SpanContext fromByteArray(byte[] bytes) {
       Utils.checkNotNull(bytes, "bytes");
-      return SpanContext.BLANK;
+      return SpanContext.getBlank();
```

Comment:
Please file an issue about the name of this, not convinced blank is the right name.

Reply:
Filed https://github.com/open-telemetry/opentelemetry-java/issues/380. I think `invalid` is a better name.